### PR TITLE
Created the initial version for Cognitive Services Batch Speech-to-te…

### PR DESCRIPTION
**Pull request by RichardSunMS**
CognitiveServicesSpeechToText connector.
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/CognitiveServicesSpeechToText/Readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Azure Cognitive Services Batch Speech-to-text Connector
```
Comment (Amjed-Ayoub) [inline certified-connectors/CognitiveServicesSpeechToText/Readme.md:1]: @RichardSunMS please follow our template for the readme file. you can find an example [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and for raw view for code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md). Please include all required sections.

File: certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1952 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CognitiveServicesSpeechToText",
+    "description": "Azure Cognitive Services for Batch Speech-to-text",
+    "version": "1.0",
+    "contact": {
+      "name": "Cognitive Services - Speech Services",
+      "url": "https://docs.microsoft.com/azure/cognitive-services/speech-service/support"
```
Comment (Amjed-Ayoub) [inline certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json:9]: @RichardSunMS please add "email" property. a support email contact.

File: certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1952 @@
...
+    }
+  },
+  "host": "eastus.api.cognitive.microsoft.com",
```
Comment (Amjed-Ayoub) [inline certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json:12]: @RichardSunMS the host URL should be in production. can you please confirm if that is the case here?

File: certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,1952 @@
...
+    "https"
+  ],
+  "paths": {
+    "/transcriptions": {
+      "get": {
+        "tags": [
+          "Custom Speech transcriptions:"
+        ],
+        "description": "Gets a list of transcriptions for the authenticated subscription.",
+        "operationId": "Transcriptions_List",
```
Comment (Amjed-Ayoub) [inline certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json:24]: @RichardSunMS all operationIds should be in Camel case. please change to "TranscriptionsList". Please fix all operationIds in your swagger file. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)

File: certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,1952 @@
...
+        "summary": "Gets a list of transcriptions for the authenticated subscription",
```
Comment (Amjed-Ayoub) [inline certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json:25]: @RichardSunMS "summary" should be different than the description. please fix all summaries and descriptions in your swagger file. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)

File: certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,1952 @@
...
+          {
+            "$ref": "#/parameters/skipQueryParameter"
+          },
+          {
+            "$ref": "#/parameters/topQueryParameter"
+          },
+          {
+            "in": "query",
+            "name": "filter",
+            "description": "A filtering expression for selecting a subset of the available transcriptions.\r\n            <ul><li><b>Supported properties:</b> displayName, description, createdDateTime, lastActionDateTime, status, locale.\r\n              </li><li><b>Operators:</b>\r\n                - eq, ne are supported for all properties.\r\n                - gt, ge, lt, le are supported for createdDateTime and lastActionDateTime.\r\n                - and, or, not are supported.\r\n              </li><li><b>Example:</b>```filter=createdDateTime gt 2022-02-01T11:00:00Z```</li></ul>",
```
Comment (Amjed-Ayoub) [inline certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json:39]: @RichardSunMS can you please explain the empty spaces in the description?

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json (hunk 3 above)
Comment (RichardSunMS) [reply certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json:12]: @Amjed-Ayoub , yes I confirm this host is already in production. But one quick question here: we will have different regional hosts here, how to make this as a parameter in future?

File: certified-connectors/CognitiveServicesSpeechToText/Readme.md (hunk 1 above)
Comment (RichardSunMS) [reply certified-connectors/CognitiveServicesSpeechToText/Readme.md]: Thanks and updated.

File: certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json (hunk 3 above)
Comment (Amjed-Ayoub) [reply certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json:13]: @RichardSunMS thank you so much for the confirmation. 
Regarding your question please see how to [Set Host URL](https://learn.microsoft.com/en-us/connectors/custom-connectors/policy-templates/dynamichosturl/dynamichosturl)

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @RichardSunMS,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**